common.mk

```diff
@@ -13,3 +13,6 @@ install-metalinter:
 	@which gometalinter > /dev/null || (go get -u github.com/alecthomas/gometalinter && \
 		go install github.com/alecthomas/gometalinter && gometalinter --install)
 	@which gometalinter > /dev/null || (echo "gometalinter install failed" && exit 1)
+
+install-linter-maptime:
+	@which maptime > /dev/null || (go get -u github.com/m3db/build-tools/linters/maptime && go install github.com/alecthomas/gometalinter)
```

Comment:
why install gometalinter too?

Reply:
bad copy pasta, meant to install maptime